<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="maVue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="maVue2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="maVue3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="maVue4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLnacerAnime">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>540</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancer Anime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonJouerSon">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>540</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Jouer le son</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChoisirFichier">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>540</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir Fichier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
